suites/manage/ocp/pvc/test_pvc.py

```diff
@@ -0,0 +1,135 @@
+"""
+PVC test plan
+"""
+import os
+import logging
+import yaml
+import pytest
+
+import suites.helpers as helpers
+import oc.openshift_ops
+
+
+ocp = oc.openshift_ops.OCP()
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def create_ceph_block_pool(request):
+    """
+    Create a Ceph block pool
+    """
+
+    def finalizer():
+        """
+        Delete the Ceph block pool created during setup
+        """
+        pass
+
+    pool_body = os.path.join("templates/ocs-deployment", "CephBlockPool.yaml")
```

Comment:
This can be used now and the yaml is loaded in the function: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs/rook.py#L144